kubernetes/autoscaling_v1alpha1_hvpa.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: autoscaling.k8s.io/v1alpha1
+kind: Hvpa
+metadata:
+  # Uncomment below annotation if you want kube-controller-manager to act on HPA's recommendation instead of hvpa-controller
```

Comment:
Might be better to clean this up and introduce it only when we have enhanced kcm?